index.html

```diff
@@ -200,53 +200,22 @@ <h1>About me</h1>
                         </ul>
                     </li>
                 </ul>
-                <!--<div class="tech">
-                    <div class="openable">
-                        <p>Languages</p>
-                        <img src="images/Techs/icons8-arrow-down-50.png">
-                    </div>
-                    <div class="techflex">
-                        <div class="stackNotice">
-                            <img src="images/Techs/icons8-javascript-48.png">
-                            <p>JavaScript</p>
-                        </div>
-                        <div class="stackNotice">
-                            <img src="images/Techs/icons8-html5-48.png">
-                            <p>HTML</p>
-                        </div>
-                        <div class="stackNotice">
-                            <img src="images/Techs/icons8-css3-48.png">
-                            <p>CSS</p>
-                        </div>
-                    </div>
-                </div>
-                <hr class="TechSeparation">
-                <div class="tech">
-                    <div class="openable">
-                        <p>Frameworks</p>
-                        <img src="images/Techs/icons8-arrow-right-50.png">
-                    </div>
-                </div>
-                <hr class="TechSeparation">
-                <div class="tech">
-                    <div class="openable">
-                        <p>Skills</p>
-                        <img src="images/Techs/icons8-arrow-right-50.png">
-                    </div>
-                </div>
-                <hr class="TechSeparation">-->
+
             </div>
```

Comment:
[OPTIONAL] I think it would be a good idea to have the technologies from your multi-post stories card show in a row as in the template, and not in a column:

Figma template:
![multipost](https://github.com/alexansaa/Personal_Portafolio/assets/111214189/877ab3f1-8c9b-4b50-b89a-7cef884bd2d6)

Your design:
![multi2](https://github.com/alexansaa/Personal_Portafolio/assets/111214189/6ba8e0df-e9fd-4292-88e5-52b4b5616efe)

Kindly make sure to stick to the same technologies mentioned in the template (Bootstrap, Ruby, etc...)
This wasn't mentioned by your previous reviewer, but I **highly** recommend you fix it. That way, your project is going to look more professional and shows that you can replicate a Figma template 👍 .

Reply:
I was checking the project corresponding to the implementation of the mentioned section. Its guidelines stated to use the tech list given there and not the ones in the Figma design. I kindly request sticking to the project requirements for having a better compliance origin and report. Maybe we can highlight the tech list defined in the project so that we comply with the requested guidelines. Figma listing may not be taken into account only its design.

Reply:
@alexansaa  I understand. If that's the case please ignore changing the technologies. As this is an OPTIONAL change, it won't prevent you from approving this project. Let me know if you need help with anything 👍